.holo/branches/emergence-site/_slate-sbg.toml

```diff
@@ -0,0 +1,9 @@
+[holomapping]
+files = [
+    "*/**",
+    "sencha-workspace/packages/**",
+    "sencha-workspace/SlateStandardsBasedGradingTeacher/**",
```

Comment:
Aren't these two lines redundant with the wildcard above? This file should probably mostly match the one above it